### ISSUE TITLE
Possibility to make a complete backup of the project

### ISSUE THREAD

**Issue by martin-hajek** [RE-QDA/requal#135]
It would be useful to have the possibility to create a complete "backup" of the project in the online mode. At present, it is possible to separately download codes, segments, memos, log, but not the entire set. I suggest creating a button on the project screen that exports all the tables in csv format and zip them into one file. The button will be functional for a user with permission to change permissions, as it corresponds to the role of the project admin.

**Comment by hlageek** [comment]
@ndporter yes, REFI-QDA is definitely on the roadmap, though import will likely be implemented before export. Unfortunately, as REFI-QDA is a minimal interoperability standard, it cannot be used as full back up, as we rely on some features that cannot be fully encoded and recovered from REFI-QDA. This concerns especially user management in the server mode. For local, single-user mode, back up is as simple as a making a copy of the project database file.
